proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -676,8 +676,7 @@ message NexusOperationInfo {
     // Operation name.
     string operation = 3;
 
-    // If set the state machine will remove itself from the HSM tree on completion.
-    bool delete_on_completion = 4;
```

Comment:
I'd just add a comment saying what was here and why it's reserved for future context.